### PR TITLE
fix: update deps

### PR DESCRIPTION
**Pull request by schuettc**
Changes:

- Update dependencies to fix deployment
- Move NestedStacks to Constructs
- Change regex in Receive Digits to collect 8 & 9

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: lib/chime-sdk-stack.ts
Comment (scottrfrancis) [inline lib/chime-sdk-stack.ts:25]: is the smaLambdaEndpointArn presence actually guaranteed? What behavior do you want if that prop doesn't exist?

File: src/lambda_function.py
Comment (scottrfrancis) [inline src/lambda_function.py:18]: a simpler construction here would be to use 'get'
dial_number = os.environ.get('DIAL_NUMBER', False)

could do same for WAVFILE_BUCKET... if it's not set line 17 will throw an Exception

File: src/lambda_function.py
Comment (scottrfrancis) [inline src/lambda_function.py:36]: can you turn this big if/else chain into a dispatch table?

**Review by scottrfrancis — COMMENTED** [review]
please consider addressing the minor comments before merging.

File: lib/chime-sdk-stack.ts
Comment (schuettc) [reply lib/chime-sdk-stack.ts:25]: Yes, for certain.  This stack will fail entirely without that.  There is no recovery.

File: src/lambda_function.py
Comment (schuettc) [reply src/lambda_function.py]: Changed.

**Review by garnathan — APPROVED** [review]
Approved, but if that double back slash in the regex is a mistake, just do a follow up CR to address.

**Pull request merged**
